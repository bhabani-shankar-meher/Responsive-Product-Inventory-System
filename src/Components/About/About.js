import React from "react";
import Box from "@mui/material/Box";
import Grid from "@mui/material/Grid";
import Divider from "@mui/material/Divider";
import Typography from "@mui/material/Typography";
const About = () => {
    return (
        <Box sx={{ width: "80%", marginLeft: "10%" }} mt={5} mb={3}>
            <Grid
                container
                rowSpacing={3}
                columnSpacing={{ xs: 1, sm: 2, md: 3 }}
            >
                <Grid item xs={12} md={12}>
                    <Typography textAlign="center" variant="h4">
                        PRODUCT INVENTORY
                    </Typography>
                    <Divider sx={{ width: "100%", marginTop: "20px" }} />
                </Grid>
                <Grid item xs={12} md={12}>
                    <Typography textAlign="center" variant="h6">
                        Product Inventory System is React CAPSTONE Project developed with functional components.
                    </Typography>
                </Grid>
            </Grid>
        </Box>
    );
};
export default About;