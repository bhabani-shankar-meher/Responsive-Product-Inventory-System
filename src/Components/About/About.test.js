import About from "./About";
import { render, screen } from "@testing-library/react";
import { create } from "react-test-renderer";

describe("ABOUT PAGE SNAPSHOT", () => {
    test("About Page Snapshot Executed Successfully", () => {
        const aboutSnapshot = create(<About />);
        expect(aboutSnapshot).toMatchSnapshot();
    });
});
describe("ABOUT PAGE TEST", () => {
    test("About Page Heading Test", () => {
        render(<About />);
        expect(
            screen.getByRole("heading", {
                name: "PRODUCT INVENTORY",
            })
        ).toBeInTheDocument();
    });
    test("About Page Test_1", () => {
        render(<About />);
        expect(
            screen.getByRole("heading", {
                name: "Product Inventory System is React CAPSTONE Project developed with functional components.",
            })
        ).toBeInTheDocument();
    });
});