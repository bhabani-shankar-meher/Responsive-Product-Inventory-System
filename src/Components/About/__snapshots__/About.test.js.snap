// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ABOUT PAGE SNAPSHOT About Page Snapshot Executed Successfully 1`] = `
<div
  className="MuiBox-root css-1mo2yih"
>
  <div
    className="MuiGrid-root MuiGrid-container css-qwfiov-MuiGrid-root"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-12 css-1idn90j-MuiGrid-root"
    >
      <h4
        className="MuiTypography-root MuiTypography-h4 css-149xwtl-MuiTypography-root"
      >
        PRODUCT INVENTORY
      </h4>
      <hr
        className="MuiDivider-root MuiDivider-fullWidth css-1dh14ht-MuiDivider-root"
      />
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-12 css-1idn90j-MuiGrid-root"
    >
      <h6
        className="MuiTypography-root MuiTypography-h6 css-1kgxqm0-MuiTypography-root"
      >
        Product Inventory System is React CAPSTONE Project developed with functional components.
      </h6>
    </div>
  </div>
</div>
`;
